doc/luigi_patterns.rst

```diff
@@ -118,6 +118,87 @@ can have implications in how the scheduler and visualizer handle task instances.
 
 	luigi RangeDaily --of MyTask --start 2014-10-31 --MyTask-my-param 123
 
+Batching mulitple parameter values into a single run
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes it'll be faster to run multiple jobs together as a single
+batch rather than running them each individually. When this is the case,
+you can mark some parameters with a batch_method in their constructor
+to tell the worker how to combine multiple values. One common way to do
+this is by simply running the maximum value. This is good for tasks that
+overwrite older data when a newer one runs. You accomplish this by
+setting the batch_method to max, like so:
+
+.. code-block:: python
+
+    class ExampleTask(luigi.Task):
+        date = luigi.DateParameter(batch_method=max)
+
+What's exciting about this is that if you send multiple ExampleTasks to
+the scheduler, it can combine them and return one. So if
+ExampleTask(date=2016-07-28), ExampleTask(date=2016-07-29) and
```

Comment:
By the way this is how it looks rendered (almost, it's not identical to readthedocs) https://github.com/Houzz/luigi/blob/985522850ecfc8d84b9e56cbeb67bde95042dfcc/doc/luigi_patterns.rst